src/main.jsx

```diff
@@ -2,9 +2,25 @@ import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App.jsx'
 import './index.css'
+import { BrowserRouter, Route, Routes } from 'react-router-dom'
+import Home from './components/Home';
+import Products from './components/Products';
+import ProductDetail from './components/ProductDetail';
 
 ReactDOM.createRoot(document.getElementById('root')).render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<App />} >
+            <Route index element={<Home />} />
+            <Route path="home" element={<Home />} />
+            <Route path="products" element={<Products />} />
+            <Route path="product-detail" element={<ProductDetail />} >
```

Comment:
Just a tip for naming convention for your URL:

If you have a list of products at `/products`, then the way you would access a single product would be to append the product ID to the next URL segment `/products/123`

This is a short resource that explains this more https://restfulapi.net/resource-naming/